### DISCUSSION TITLE
airflow moitoring with new relic

### DISCUSSION THREAD

**Discussion by NonlinearNimesh** [apache/airflow#27801] [Q&A]
Hi, i wanted to monitor airflow with new relic, can anyone please tell me the is it possible or not, if yes then how can i do it.
i found this (https://github.com/newrelic/newrelic-airflow-plugin) link but i failed to integrate new relic with airflow after following the not so clear instruction, 

Can anyone help me regarding this.

Thanks

**Comment by Taragolis** [comment]
@NonlinearNimesh seems like plugin not updated for about 1.5 year so it possible that this plugin doesn't work correctly with new version of Airflow (just assumption I've newer use this plugin).

Airflow have ability to send monitoring into [StatsD](https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/metrics.html) and New Relic has integration [StatsD to New Relic](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/statsd-monitoring-integration-version-2/#install) which might be another option to send metrics to NR

**Comment by potiuk** [comment]
We also have https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-49+OpenTelemetry+Support+for+Apache+Airflow which is approved and just needs to get implemented. If anyone (including you @NonlinearNimesh would like to be involved or lead the implemnentation - it's always welcome).
